examples/README.md

```diff
@@ -0,0 +1,11 @@
+Python wrapper examples
+-----------------------
+
+In this part, you can find and share real use cases for the OVH python wrapper
+
+## OVH services
+
+Following examples are related to cross services proposed by OVH.
```

Comment:
Not sure to understand. Do you mean that the following examples are not tied to any spacific OVH service and try to gather / centralize informations ?